app/modules/integrations/integrations_service.py

```
@@ -2324,3 +2428,351 @@ async def deactivate_jira_integrations_for_user(self, user_id: str) -> int:
+        from .token_encryption import decrypt_token
+
+        db_integration = (
+            self.db.query(Integration)
+            .filter(Integration.integration_id == integration_id)
+            .filter(Integration.integration_type == IntegrationType.CONFLUENCE.value)
+            .filter(Integration.active == True)  # noqa: E712
+            .first()
+        )
+
+        if not db_integration:
+            raise Exception(f"Confluence integration {integration_id} not found")
```

Ruff (0.14.3)

Create your own exception

(TRY002)